blueprints/app/files/package.json

```diff
@@ -77,21 +79,24 @@
     "ember-source": "~6.0.0-beta.1",
     "ember-template-lint": "^6.0.0<% if (welcome) { %>",
     "ember-welcome-page": "^7.0.2<% } %>",
-    "eslint": "^8.57.1",
+    "eslint": "^9.12.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-ember": "^12.2.1",
-    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-n": "^17.11.1",
     "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-qunit": "^8.1.2",
+    "globals": "^15.11.0",
     "loader.js": "^4.7.0",
     "prettier": "^3.3.3",
+    "prettier-plugin-ember-template-tag": "^2.0.2",
```

Comment:
seem like this is new here, don't you need to configure it in `.prettierrc.js` per https://github.com/gitKrystan/prettier-plugin-ember-template-tag?tab=readme-ov-file#usage?

Reply:
Looks like what I have work in neovim, but not in vscode.

ugh. thanks for double checking

Reply:
fixed!